test/lightning/version_control/version_control_usage_limiter_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Lightning.VersionControl.VersionControlUsageLimiterTest do
+  use ExUnit.Case, async: true
+
+  alias Lightning.Extensions.MockUsageLimiter
+  alias Lightning.Extensions.Message
+  alias Lightning.Extensions.UsageLimiting.Action
+  alias Lightning.Extensions.UsageLimiting.Context
+  alias Lightning.VersionControl.VersionControlUsageLimiter
+
+  describe ".limit_github_sync/1" do
```

Comment:
Why the `.` ? Is this some sort of naming standard ?

Reply:
@elias-ba Oops - that is a habit from Ruby tests - '.' indicates a class-level method as opposed to '#' which indicates an instance-level method. Have corrected in a follow-up PR.